source/material_model/visco_plastic.cc

```diff
@@ -661,7 +661,7 @@ namespace aspect
           // If plastic strain is tracked (so not the total strain), only overwrite
           // when plastically yielding.
           // If viscous strain is also tracked, overwrite the second reaction term as well.
-          if  (this->get_timestep_number() > 0 && in.strain_rate.size())
+          if  (this->get_timestep_number() != numbers::invalid_unsigned_int && this->get_timestep_number() > 0 && in.strain_rate.size())
```

Comment:
is this equivalent to `this->simulator_is_past_initialization() && this->get_timestep_number() > 0 && in.strain_rate.size())`? That seems a bit easier to understand.

Reply:
Yes, I think so. I will change it.

Reply:
Thanks, @gassmoeller, for seeing this. I had noticed it too but forgot to mark it up.